relay-config/src/redis.rs

```diff
@@ -193,6 +217,14 @@ pub enum RedisConfigRef<'a> {
         /// Options of the Redis config.
         options: RedisConfigOptions,
     },
+
+    Sentinel {
+        /// Reference to the Redis nodes urls of the cluster.
+        sentinel_nodes: &'a Vec<String>,
+        master_name: &'a String,
```

Comment:
Types like `&Vec<T>` and `&String` should be avoided, use `&[T]` and `&str` instead:
```suggestion
        sentinel_nodes: &'a [String],
        master_name: &'a str,
```